### PR TITLE
Typography pass

### PR DESCRIPTION
**Pull request by jckr**
Screen Shot 2019-08-09 at 2 00 08 PM
Screen Shot 2019-08-09 at 1 59 58 PM

### CODE REVIEW

File: modules/gatsby/bin/ocular.js [hunk 1]
```diff
@@ -207,28 +206,27 @@ const commands = {
 
   help() {
     console.log(`
-Ocular CLI
-----------
+    Ocular CLI
```
Comment (ibgreen) [inline modules/gatsby/bin/ocular.js]: Maybe a string constant?

Indentation looks a little strange?

File: modules/gatsby/src/components/styled/typography.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
+import {styled} from 'baseui';
+
+// h4, h5, and h6 haven't been spec'ed yet, so we'll just reuse baseweb defaults
+export {H4, H5, H6} from 'baseui/typography';
+
+const monospaceFontStack = `'SF Mono', Consolas, Menlo, Monaco, 'Andale Mono WT', 'Andale Mono', 'Lucida Console', 'Lucida Sans Typewriter', 'DejaVu Sans Mono', 'Bitstream Vera Sans Mono', 'Liberation Mono', 'Nimbus Mono L', 'Courier New', Courier, monospace`;
```
Comment (ibgreen) [inline modules/gatsby/src/components/styled/typography.js]: What does this do? Various fallbacks for the fixe width font? Worth a comment for those of us who are not baseui/styling fiends?

File: modules/gatsby/src/gatsby-node/create-webpack-config.js [hunk 3]
```diff
@@ -1,5 +1,7 @@
   const {log, COLOR} = require('../utils/log');
 
+  const MODULE_NAME = '@jckr/ocular-gatsby';
```
Comment (ibgreen) [inline modules/gatsby/src/gatsby-node/create-webpack-config.js]: ??? Change before landing?

File: modules/gatsby/src/templates/doc-page-markdown.jsx [hunk 4]
```diff
@@ -1,13 +1,51 @@
 import React from 'react';
-
+import rehypeReact from 'rehype-react';
 import {graphql} from 'gatsby';
 
+// note - these typographic elements are taken directly from baseui.
+// we can consider customizing them by first importing in styled/index, then
+// giving them special parameters
+
+import {CodeBlock, H1, H2, H3, H4, H5, H6, InlineCode, P, Pre, MarkdownBody} from '../components/styled/';
```
Comment (ibgreen) [inline modules/gatsby/src/templates/doc-page-markdown.jsx]: Remove trailing slash from import?

File: modules/gatsby/src/templates/doc-page-markdown.jsx [hunk 5]
```diff
@@ -20,39 +58,35 @@ export const query = graphql`
...
-  });
-}
-
 export default class DocTemplate extends React.Component {
   constructor(props) {
     super(props);
-    this.state = {html: replaceLinks(props)};
+    const {relativeLinks} = props.pageContext;
+    // note - we can add many other custom components.
+    const renderAst = new rehypeReact({
```
Comment (ibgreen) [inline modules/gatsby/src/templates/doc-page-markdown.jsx]: Care to add a brief comment about what rehypeReact does here? I am not familiar with the component

File: modules/gatsby/src/gatsby-node/create-webpack-config.js (hunk 3 above)
Comment (jckr) [reply modules/gatsby/src/gatsby-node/create-webpack-config.js]: thanks for picking this up!!! yes this is to make it easier to test w/ local changes

**Comment by jckr** [comment]
This represents an effort in the right direction however this really breaks the layout for mobile widths. So I don't recommend we merge it before we have a solution for mobile sites.

**Pull request merged**